proposals/description/deprecate_incentives.ts

```diff
@@ -0,0 +1,212 @@
+    },
+    {
+      target: 'stakingTokenWrapperBribe3Crvpool',
+      values: '0',
+      method: 'harvest()',
+      arguments: (addresses) => [],
+      description: `
+      Harvest the Votium bribes: 3crv-FEI metapool staking token wrapper to fund deposit. This will
+      harvest 1.5M TRIBE and send it to the votiumBriber contract for the 3Crv-pool. This will then later be harvested.
+
+      Harvested here to remove any possibility that the harvest is conducted during the DAO vote and the withdrawal amount is 
+      incorrect. Later withdrawn from the VotiumBriber contract.
+      `
+    },
+    ////////////   END OF PERFORM OUTSIDE OF PROPOSAL FIRST  ///////////////////
+
+    // Withdraw excess TRIBE from reward system
+    {
+      target: 'erc20Dripper',
+      values: '0',
+      method: 'withdrawERC20(address,address,uint256)',
+      arguments: (addresses) => [addresses.tribe, addresses.core, '2508838570600494412126519'],
+      description: 'Withdraw 2.5M TRIBE from the ERC20 Dripper to the Core Treasury'
+    },
+    {
+      target: 'votiumBriber3Crvpool',
+      values: '0',
+      method: 'withdrawERC20(address,address,uint256)',
+      arguments: (addresses) => [addresses.tribe, addresses.core, '1725076846586787179639648'],
+      description: `
+      Withdraw all TRIBE from 3CRV Votium briber contract to the Core Treasury. 
+      
+      Withdraw is made up of 200k TRIBE pre-existing on this contract and then an additional 1.5M TRIBE 
+      that was harvested from the stakingTokenWrapperBribe3Crvpool.
+      `
+    },
+    {
+      target: 'votiumBriberD3pool',
+      values: '0',
+      method: 'withdrawERC20(address,address,uint256)',
+      arguments: (addresses) => [addresses.tribe, addresses.core, '232096077769383622085234'],
+      description: `
+      Withdraw all TRIBE from D3 Votium briber contract to the Core Treasury. 
+      
+      Withdraw is made up of 230k TRIBE that was harvested from the stakingTokenWrapperBribeD3pool.
+      `
+    },
+    {
+      target: 'rewardsDistributorAdmin',
+      values: '0',
+      method: '_grantComp(address,uint256)',
+      arguments: (addresses) => [addresses.core, '164000000000000000000000'],
+      description: `Withdraw excess 164k TRIBE from Rari delegator contract`
+    },
+    {
+      target: 'tribalChief',
+      values: '0',
+      method: 'governorWithdrawTribe(uint256)',
+      arguments: (addresses) => ['26833947775112516867325654'], // TODO: Update if do the harvests before this script
+      description: `
+      Withdraw remaining TRIBE from the Tribal Chief to the Core Treasury. 
+      
+      Withdrawal amount = 
+      (Tribal Chief balance before harvest 
+          - staking token wrapper harvested TRIBE
+            - (pending rewards, Uniswap-v2 FEI/TRIBE LP + Curve 3crv-FEI metapool LP + G-UNI DAI/FEI 0.05% fee tier)
+      
+      Withdrawal amount = 29.2M - 1.8M - 565k = 26.8M
```

Comment:
Staking token wrapper balance = constant
Tribal Chief starting balance may decrease, in which case the pending rewards for LPs will decrease by the same amount